PF-build.sh

```diff
@@ -110,74 +157,223 @@ if ! type zip > /dev/null; then
 		echo "Run git Bash under Administrator privilege and"
 		echo "navigate to the directory /c/Program Files/Git/mingw64/bin,"
 		echo "you can run the command $(tput setaf 2)ln -s /c/Program Files/7-Zip/7z.exe zip.exe$(tput sgr0)"
-		exit
+		exit 3
 	elif [ $OSTYPE == "linux-gnu" ]; then
 		echo "$(tput setaf 1)Missing 'zip' which is important to run this script"
 		echo "install it with the command $(tput setaf 2)'sudo apt-get install zip'$(tput sgr0)"
-		exit
+		exit 3
+	fi
+fi
+# Check python ... needed during language build
+if ! type python > /dev/null; then
+	if [ $OSTYPE == "msys" ]; then
+		echo "$(tput setaf 1)Missing 'python' which is important to run this script"
+		exit 4
+	elif [ $OSTYPE == "linux-gnu" ]; then
+		echo "$(tput setaf 1)Missing 'python' which is important to run this script"
+		echo "As Python 2.x will not be maintained from 2020 please,"
+		echo "install it with the command $(tput setaf 2)'sudo apt-get install python3'."
+		echo "Check which version of Python3 has been installed using 'ls /usr/bin/python3*'"
+		echo "Use 'sudo ln -sf /usr/bin/python3.x /usr/bin/python' (where 'x' is your version number) to make it default.$(tput sgr0)"
+		exit 4
 	fi
 fi
-###End prepare bash enviroment
 
-BUILD_ENV="1.0.2"
+#### End prepare bash environment
+
+
+#### Set build environment 
+ARDUINO_ENV="1.8.5"
+BUILD_ENV="1.0.6"
+BOARD="PrusaResearchRambo"
+BOARD_VERSION="1.0.1"
+BOARD_URL="https://raw.githubusercontent.com/DRracer/Arduino_Boards/master/IDE_Board_Manager/package_prusa3d_index.json"
+BOARD_FILENAME="prusa3drambo"
+BOARD_FILE_URL="https://raw.githubusercontent.com/DRracer/Arduino_Boards/master/IDE_Board_Manager/prusa3drambo-1.0.1.tar.bz2"
+PF_BUILD_FILE_URL="https://github.com/3d-gussner/PF-build-env/releases/download/$BUILD_ENV/PF-build-env-$BUILD_ENV.zip"
+LIB="PrusaLibrary"
 SCRIPT_PATH="$( cd "$(dirname "$0")" ; pwd -P )"
 
 # List few useful data
 echo
-echo "Script path:" $SCRIPT_PATH
-echo "OS         :" $OS
-echo "OS type    :" $OSTYPE
+echo "Script path :" $SCRIPT_PATH
+echo "OS          :" $OS
```

Comment:
Unused variable? Yields empty string for all tested scenarios.

Reply:
Which variable is unused?
Here my results with `git-bash`
```
$ bash PF-build.sh
Windows 64-bit found

Script path : <secret path>/Prusa-Firmware/MK3_PF-build_106
OS          : Windows_NT
OS type     : msys

Ardunio IDE : 1.8.5
Build env   : 1.0.6
Board       : PrusaResearchRambo
Specific Lib: PrusaLibrary
```

Reply:
Yes, ok. I have overlooked that Windows use $OS. It is empty just on Linux-based systems.